draft-ma-opsawg-ucl-acl.md

```diff
@@ -115,32 +115,36 @@ informative:
       current context, and map the users to their correct access
       entitlement to the network.
 
-   This document defines a common schedule YANG module which is designed
-   to be applicable for policy activation based on date and time
-   conditions. This model is designed with the intent to be reusable in other scheduling contexts.
+   This document defines a common schedule YANG module in {{sec-schedule}} which
+   is designed to be applicable for policy activation based on date and time
+   conditions. This model is designed with the intent to be reusable in other
+   scheduling contexts.
```

Comment:
```suggestion
   scheduling contexts. In particular, the groupings can be reused by other modules.
```

Reply:
Please fix the pyang issue (see the report at https://github.com/boucadair/policy-based-network-acl/actions/runs/5077408435/jobs/9120637480)

Reply:
Fixed.